core/storage/blog/gae_models.py

```diff
@@ -14,7 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Models for storing data related to blog posts."""
+"""Models for storing data related to blog posts. The author can edit both
+published and draft blog post at any time. Hence, functionality for reverting
```

Comment:
This explanation should probably go in the BlogPostModel docstring (to justify why it inherits from BaseModel rather than VersionedModel).

I also don't quite understand the explanation. What do you mean by "the author can edit both published and draft blog post at any time" and why is this relevant? I mean ... authors can edit published and draft explorations at any time, right? The explanation should be clear about what the difference is compared to similar content models which use VersionedModel instead (so, in particular, the points mentioned here should not apply to those models as well).

Reply:
Done. The only reason BlogPostModel is not versioned is because we don't have any plans to support reverting back to earlier versions (discussed during GSoC'21)